### PR TITLE
Clean up admin UI

### PR DESCRIPTION
**Pull request by thutterer**
This PR gives the Admin UI some love.
It might not be the most important area to improve but it is the first thing you see when you set up your dev env as a new contributor to the project.

Hope my commit messages explain the what and why well enough :nerd_face:

### CODE REVIEW

**Comment by DavidKang** [comment]
@thutterer, could you add some screenshot? :smile_cat:

File: app/controllers/episodes_controller.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@ def create
     @episode = Episode.new(episode_params)
 
     if @episode.save
-      redirect_to episode_path(@episode), notice: 'Episode was successfully created.'
+      redirect_to episodes_path, notice: 'Episode was successfully created.'
```
Comment (DavidKang) [inline app/controllers/episodes_controller.rb]: Why are you redirecting to the index view?

Comment (thutterer) [reply app/controllers/episodes_controller.rb]: See commit message comment in 0b88a5b

**Comment by thutterer** [comment]
@DavidKang I updated the specs and rebased the branch on the latest master which had your fix for the "piwik vs tests" issue.
So you have to delete your local copy of the branch and pull again before you can give this another test ride :car:

**Pull request merged**